apps/api/src/server/applications/application/application.validators.js

```diff
@@ -34,16 +34,20 @@ const validateExistingRegions = async (value) => {
 	}
 };
 
-/**
- *
- * @param {number} caseId
- */
-const validateExistingApplication = async (caseId) => {
+/** @type {import('express').RequestHandler } */
+export const validateExistingApplication = async (request, response, next) => {
```

Comment:
we exported it to use it in the routes so that it can return error 404, it seems using the previous validation function syntax only allowed for error 400..this was to get around that